hazelcast/src/test/java/com/hazelcast/monitor/TimedMemberStateTest.java

```diff
@@ -80,4 +85,10 @@ public void testReplicatedMapGetStats() {
         ReplicatedMapService replicatedMapService = nodeEngine.getService(ReplicatedMapService.SERVICE_NAME);
         assertNotNull(replicatedMapService.getStats().get("replicatedMap"));
     }
+
+    @Test
+    public void testMapPutAllStats() {
+        LocalMapStatsImpl localMapStats = (LocalMapStatsImpl) timedMemberState.getMemberState().getLocalMapStats("name");
```

Comment:
I believe, to run some `TimedMemberStateTest ` tests on multiple nodes and to assert results according to each node, if possible, will make tests more reliable.